functions/Get-DbaClusterSqlInstance.ps1

```diff
@@ -0,0 +1,84 @@
+function Get-DbaClusterSqlInstance
+{
+<#
+.SYNOPSIS
+Returns the registered SQL Server Instances on a windows Server cluster
+	
+.DESCRIPTION
+By default, this command returns for each Cluster passed in:
+Pulls Cluster details via WMI/CIM so doesn't require Failover Cluster PowerShell module to be installed.
+Multiple clusters can be piped in.
+Returns ALL instance registered on cluster. Does not check that they are viable, correctly installed or currenly running. 
+	
+.PARAMETER Cluster
+The SQL Server that you're connecting to.
+
+.PARAMETER ClusterCredential
```

Comment:
To make it all more consistent, we chose to use 'Credential' for anything that isn't 'SqlCredential'.
Of course, there may be exceptions.
Like adding 'source' and 'destination' if we need two for the copy- functions, for example.
I guess in this case, there can't be much confusion?